core/js/src/main/scala/zio/internal/DefaultExecutors.scala

```diff
@@ -18,4 +18,6 @@ package zio.internal
 
 import zio.stacktracer.TracingImplicits.disableAutoTrace
 
-private[zio] abstract class DefaultExecutors
+private[zio] abstract class DefaultExecutors {
+  final def signalBlocking(): Unit = ()
```

Comment:
doesn't it make more sense to provide a scoped API, something similar to `scala.concurrent.blocking`

Reply:
As I mentioned in the other comment, I don't think we can scope it. We're not shifting execution of the runnable to another threadpool, we're removing the current thread altogether from the current threadpool and replace it with another one. The new thread doesn't do anything yet, we do this only so that we don't exhaust the bounded threadpool and end up in a deadlock

Reply:
Although I think this method is better placed in the `Blocking` object now that you mentioned it